web/collections.textile

```diff
@@ -213,7 +233,7 @@ or pass in a function (the Scala compiler automatically converts our method to a
 scala> def timesTwo(i: Int): Int = i * 2
 timesTwo: (i: Int)Int
 
-scala> numbers.map(timesTwo)
+scala> numbers.map { timesTwo }
```

Comment:
technically our style guide says that numbers.map(timesTwo) is correct because it is a method reference and not an anonymous function. There is no reason to obsess over the difference though